src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+import books from '../../components/BookItem';
+
+const initialstate = {
+    books: books
+};
+
+const bookSlice = createSlice({
+    name: 'book',
+    initialState: initialstate,
+    reducers: {
+        removeItem: (state, action) => {
+            const itemId = action.payload;
+            state.books = state.books.filter((item) => item.id !== itemId);
+        }
+    }
+});
+
+export const {removeItem} = bookSlice.actions
+
```

Comment:
- [ ] After you add the `addBook` reducer function for adding books to the store, kindly ensure that you export as well :+1: